src/ResourceManager/Sql/Commands.Sql/Failover Group/Cmdlet/RemoveAzureSqlDatabaseFromFailoverGroup.cs

```diff
@@ -150,7 +162,7 @@ public List<string> ConvertDatabaseModelToDatabaseHelper(ICollection<AzureSqlDat
             {
                 if (String.Compare(model.DatabaseName, "master", StringComparison.Ordinal) == 0)
                 {
-                    WriteWarning("Can not add master database into the failover group. This operation request will be ignored");
+                    WriteWarning("Can not remove master database into the failover group. This operation request for removing master database will be ignored");
```

Comment:
Is it possible for the master to ever be in the failover group? 

also:
"...remove master database ~~into~~ __from__ the failover group".